.github/workflows/rust.yml

```diff
@@ -45,6 +45,32 @@ jobs:
           command: check
           args: --all-targets --no-default-features --workspace
 
+  wasm:
+    name: Check WASM compatibility
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install Rust stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          target: wasm32-unknown-unknown
+          override: true
+
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@v1.3.0
+
+      - name: Check all features
+        uses: actions-rs/cargo@v1.0.3
+        with:
+          command: check
+          # Test targets don't compile to WASM successfully because
+          # we run tests using BitVec<u64,_> which doesn't.
```

Comment:
is it annoying to feature gate the test using `BitVec`? :P 

I guess we could run tests using this `wasm-pack` otherwise to run tests on.

Reply:
Actually I just realised that this isn't true for `scale-value`; we don't use bitvec anywhere! `scale-bits` and `scale-decode` use bitvec in the tests and so we can't compile their tests to wasm without hiding certain things we'd rather not (any test involving a bitvec with a u64 store type).